solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -102,7 +102,10 @@ export class DataSetService implements DataSetPlugin {
   }
 
   public async removeAllAccessPermissions(datasetId: string): Promise<PermissionsResponse> {
-    const response = await this._dataSetsAuthService.removeAllAccessPermissions(datasetId);
+    const response = await this._dataSetsAuthService.removeAllAccessPermissions(datasetId, {
```

Comment:
is this just a placeholder?

Reply:
oh, the authenticatedUser? Yes! `res.locals.user` should be passed from the route handler.